src/authentication/auth-config.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
```

Comment:
I would remove this comment, we will be editing.

Reply:
kept the comment, but removed warning: do not edit